### PR TITLE
update committee

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by vchuravy** [comment]
I hope I haven't missed anybody.

**Comment by jpfairbanks** [comment]
Is the order intended to be significant? If not, then maybe:

- Exec Chair
- Committee chairs 
- Vice chairs 

in alphabetical order.

**Comment by vchuravy** [comment]
cc: @timholy

**Comment by essenciary** [comment]
Wondering if we can mention the other contributors as well, somewhere? A couple more people contributed to the new website and it would be nice to include them.

**Comment by vchuravy** [comment]
Absolutely! Credit where credit is due. Maybe a `Thank you` or `Volunteer` (or whatever wording you prefer) after the contact information?

File: _includes/content.html [hunk 1]
```diff
@@ -25,7 +25,23 @@ <h2>Call for Proposals</h2>
...
+          <li>James Fairbanks (program vice-chair)</li>
+          <li>Samuel Powell (local vice-chair)</li>
+          <li>Avik Sengupta</li>
+          <li>Eric Davies</li>
+          <li>Mike Innes</li>
+        </ul>
+        </p>
         <p>
           The JuliaCon committee is composed entirely of volunteer organizers
           and can be reached at <a href="mailto:juliacon@googlegroups.com">juliacon@googlegroups.com</a>
```
Comment (vchuravy) [inline _includes/content.html]: I thought this is `committee@juliacon.org`

**Comment by ViralBShah** [comment]
Given that @aviks has done 90% of the work on sponsorships, I think it would be best to name him finance chair.

**Comment by essenciary** [comment]
@vchuravy Can this be published? 

I'll also add the Volunteers, but for now I'm waiting for their OK to add their info.

**Pull request merged**